src/test/java/StorageTest.java

```diff
@@ -0,0 +1,34 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import hoodini.Storage;
+
+
+
+
```

Comment:
Same thing as ShuckyCheese said, I've noticed you tend to leave some lines under some code, maybe just reduce it to one line so that it would be cleaner and better?